edge/pkg/common/modules/names.go

```diff
@@ -11,4 +11,6 @@ const (
 	EdgeStreamModuleName = "edgestream"
 	// DeviceTwinModuleName name
 	DeviceTwinModuleName = "twin"
+	// MetaManagerModuleName name
+	MetaManagerModuleName = "metaManager"
```

Comment:
`EdgeStreamModuleName = "edgestream"`
Uniform case is recommended.

Reply:
we can open another PR to fix it, need to modify all "metaManager" related codes, and move this string literal to the common modules just as processor also need to use this module name. If not, compile will be failed due to `import cycle not allowed `

Reply:
okey.